Userland/Services/WebDriver/Client.cpp

```diff
@@ -538,4 +539,15 @@ ErrorOr<JsonValue, HttpError> Client::handle_delete_all_cookies(Vector<StringVie
     return make_json_value(result);
 }
 
+// POST /session/{session id}/cookie https://w3c.github.io/webdriver/#dfn-adding-a-cookie
```

Comment:
This goes before the deletion methods. I'll add section numbers to all of these later, then it becomes obvious.

Reply:
Same everywhere below